barretenberg/cpp/src/barretenberg/honk/sumcheck/relation_correctness.test.cpp

```diff
@@ -78,8 +79,8 @@ template <typename Flavor> void create_some_add_gates(auto& circuit_builder)
     }
 
     // If Ultra arithmetization, add a big add gate with use of next row to test q_arith = 2
-    if constexpr (IsUltraFlavor<Flavor>) {
-        fr e = a + b + c + d;
+    if constexpr (proof_system::IsUltraFlavor<Flavor>) {
```

Comment:
Yeah, I removed the prover_library files as they were not needed anymore and the compiler got confused without the namespace qualifiers. Those files weren't needed because the functionality is now only used by the Instance so I moved it inside the instance. (btw the master merge that moved bberg undeleted my deletes but now I've removed them again, sorry about this)